pkg/docker/docker.go

```diff
@@ -25,24 +23,28 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var (
+	Tag               string
+	VerificationImage = "cnabio/signy-in-toto-verifier:" + Tag
+)
+
 const (
-	layoutPath = "/in-toto/layout.template"
-	keyPath    = "/in-toto/key.pub"
+	// Where we expect to copy in-toto artifacts to.
```

Comment:
I would assume the linter is unhappy because the comment is not `// WorkingDir is the directory...`